src/main/java/com/programmers/ticketparis/service/schedule/ScheduleService.java

```diff
@@ -0,0 +1,51 @@
+package com.programmers.ticketparis.service.schedule;
+
+import static com.programmers.ticketparis.exception.ExceptionRule.*;
+
+import java.util.List;
+
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import com.programmers.ticketparis.domain.schedule.Schedule;
+import com.programmers.ticketparis.dto.schedule.request.ScheduleCreateRequest;
+import com.programmers.ticketparis.dto.schedule.response.ScheduleResponse;
+import com.programmers.ticketparis.exception.ScheduleException;
+import com.programmers.ticketparis.repository.schedule.ScheduleRepository;
+
+import lombok.RequiredArgsConstructor;
+
+@Service
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+public class ScheduleService {
+
+    private final ScheduleRepository scheduleRepository;
+
+    @Transactional
+    public ScheduleResponse createSchedule(Long performanceId, ScheduleCreateRequest request) {
+        // performanceId에 해당하는 공연 데이터가 없는 경우 예외 처리 예정 (2023.09.06 김영주 작성)
+        // startDatetime이 공연의 시작, 종료날짜 범위를 벗어나는 경우 예외 처리 예정 (2023.09.06 김영주 작성)
```

Comment:
`p4`
TODO 주석을 사용해보세요~

```suggestion
        // TODO: performanceId에 해당하는 공연 데이터가 없는 경우 예외 처리 예정 (2023.09.06 김영주 작성)
        // TODO: startDatetime이 공연의 시작, 종료날짜 범위를 벗어나는 경우 예외 처리 예정 (2023.09.06 김영주 작성)
```

참고 : https://cheese10yun.github.io/intellij-todo/

Reply:
반영: [313ac9a](https://github.com/prgrms-be-devcourse/BE-04-TICKETPARIS/pull/30/commits/313ac9afb4503e98fe2d153024cda2aff1dafe32)
- 이런 기능이 있는 줄 몰랐는데 정말 유용하네요! 완전 감사합니다 ㅎㅎ